website/components/LandingPage/OpenSource/index.js

```diff
@@ -37,43 +37,45 @@ function cacheKey(key) {
 function cacheGet(key) {
   try {
     const { expires, data } = JSON.parse(localStorage.getItem(cacheKey(key)));
-    if (expires > Date.now()) return data;
+    return { expired: expires < Date.now(), data };
   } catch (error) {
     // Nothing.
   }
+  return {};
```

Comment:
It'd be better to return `{ expired: true, data: null }` directly in the `catch` block.